test/correctness/simd_op_check.cpp

```diff
@@ -82,7 +82,9 @@ struct Test {
         target = get_target_from_environment()
             .with_feature(Target::NoBoundsQuery)
             .with_feature(Target::NoAsserts)
-            .with_feature(Target::NoRuntime);
+            .with_feature(Target::NoRuntime)
+            .with_feature(Target::DisableLLVMLoopUnroll)
```

Comment:
We should really make it a point to move forward into making these set by default (I presume the major obstacle being risk of injecting bugs and/or perf regressions in existing code)

Reply:
Yes, they may cause perf regressions on existing code where there's room for better scheduling (which there very often is). I enabled them here because humans read this assembly, and any automatic unrolling and vectorizing is a red herring.